Configuration/Routes.Service.yaml

```diff
@@ -31,19 +31,19 @@
   httpMethods: ['POST']
 
 -
-  name: 'Copy node to clipboard'
-  uriPattern: 'copy-node'
+  name: 'Copy nodes to clipboard'
+  uriPattern: 'copy-nodes'
```

Comment:
Isn`t that kind of breaking when we change the end-point URL, even it is makes sense?

Reply:
Those endpoints are not part of our public API, it's merely an implementation detail to be able to keep the copied nodes between different sites/domains.